src/docs/clients/csharp/index.mdx

```diff
@@ -7,6 +7,12 @@ noindex: true
 tags: []
 ---
 
+<Alert level="warning" title="Note"><markdown>
+
+For .NET Framework 4.6.1, .NET Core 2.0, Mono 5.4 or higher we recommend using the new [.NET SDK](/platforms/dotnet). Raven is still recommended for .NET Framework 3.5 to 4.6.0.
```

Comment:
@bruno-garcia does this look right to you?

Reply:
tiny nit (serial commas): 
```suggestion
For .NET Framework 4.6.1, .NET Core 2.0, Mono 5.4, or higher we recommend using the new [.NET SDK](/platforms/dotnet). Raven is still recommended for .NET Framework 3.5 to 4.6.0.
```